ctios/__init__.py

```diff
@@ -0,0 +1,398 @@
+###############################################################################
```

Comment:
@janadamek-programmer Thanks for your PR.

I suggest to avoid duplicating ctios library code here. Please remove it from this PR. Keep only changes relevant to the plugin.  The new requirement (ctios) can be explained in README.md (enough for development). For distribution (a zip file) the library will be included of course.

Moreover the code of ctios library is planned to be refactorized and improved.